lib/containers/basetest.pm

```diff
@@ -8,6 +8,7 @@
 
 package containers::basetest;
 use Mojo::Base 'opensusebasetest';
+use containers::engine;
```

Comment:
Before you ask, not sure why I removed that in the previous PR... without this, nothing works.